### PR TITLE
Make kubemark push a manual target

### PR DESCRIPTION
**Pull request by krzyzacy**
fix `bazel build //...`

/assign @ixdy @mikedanese

### CODE REVIEW

File: cluster/images/kubemark/BUILD [hunk 1]
```diff
@@ -16,6 +16,7 @@ docker_push(
     repository = "$(PROJECT)/kubemark",
     stamp = True,
     tag = "latest",
+    tags = ["manual"],
```
Comment (mikedanese) [inline cluster/images/kubemark/BUILD]: what does this do?

Comment (krzyzacy) [reply cluster/images/kubemark/BUILD]: skips the target from `bazel build //...`, for manual runs we'll `--define PROJECT=foo`

https://docs.bazel.build/versions/master/bazel-user-manual.html#target-patterns
```
Targets with tags=["manual"] will not be included in wildcard target patterns (..., :*, :all, etc). You should specify such test targets with explicit target patterns on the command line if you want Bazel to build/test them.
```

**Pull request merged**